lib/campaign/CampaignState.h

```diff
@@ -42,12 +42,19 @@ class DLL_LINKAGE CampaignRegions
 		std::string infix;
 		int xpos;
 		int ypos;
+		std::optional<int> xLabelpos;
+		std::optional<int> yLabelpos;
```

Comment:
Not sure if Point has serialize() method. If not - then easy to add anyway.
```suggestion
		Point pos;
		std::optional<Point> labelPos;
```